edi_oca/models/edi_exchange_type.py

```diff
@@ -134,12 +143,30 @@ def _check_backend(self):
             if rec.backend_id.backend_type_id != rec.backend_type_id:
                 raise exceptions.UserError(_("Backend should respect backend type!"))
 
+    def _make_exchange_filename_datetime(self):
+        """
+        Returns current datetime (now) using filename pattern
+        which can be set using advanced settings.
```

Comment:
Didn't notice this on 14.0 but not a blocker:

```suggestion
        """Returns current datetime (now) using filename pattern.
        
        The pattern can be configured using `filename_pattern` key in advanced settings.
```
Next time, don't change it now as it is the same on 14.0.
https://peps.python.org/pep-0257/#multi-line-docstrings